wp-content/themes/csisjti/template-parts/entry-header.php

```diff
@@ -12,25 +12,91 @@
 ?>
 
 <header class="single__header<?php echo esc_attr( $entry_header_classes ); ?>">
+    
+	<?php csisjti_share(); ?>
+  
+	<?php if (get_post_type() == 'event') { ?>
 
-	<div class="single__header-wrapper">
+		
+			<?php echo '<p class="single__header-category">' . csisjti_display_categories() . '</p>'; ?>
```

Comment:
Note that the `csisjti_display_categories()` returns the correct markup already, so no need to wrap it in a `<p>` tag. You'll need to update the CSS accordingly.